js-intro.js

```diff
@@ -36,8 +62,17 @@ var myNumbers2 = [8, -7, 0, 6, 2]
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+removeVowel = (str) => {
+  let newArr1 = str.split("")
+  let newArr2 = newArr1.filter(value => value.toLowerCase() !== "a" && value.toLowerCase() !== "e" && value.toLowerCase() !== "i" && value.toLowerCase() !== "o" && value.toLowerCase() !== "u")
+  let newStr = newArr2.join("")
+  return newStr
+  }
 
 
+// console.log(removeVowel(stringWithVowels1))
+// console.log(removeVowel(stringWithVowels2))
+//$$$Is there a shorter way to write this?
```

Comment:
Yes! I would do some google fu and check for "remove vowels js function." There are a lot of cool solutions out there ;)